docs/overviews/overview-publishers.md

```diff
@@ -37,17 +37,40 @@ Here are just some of the intended benefits of integrating with UID2:
 
 ## Resources
 
-The following documentation resources are available for publishers to implement UID2.
+The following documentation resources are available for publishers to implement UID2:
+- [Web Integration](#web-integration)
```

Comment:
Should we say 

_Web, Prebid and GAM Integration_

to be more descriptive?

_The following documentation resources relate to web integration for publishers._

This can stay unchanged (but up to you)

?

Reply:
Updated to: "The following documentation resources relate to web, Prebid, and Google Ad Manager integration for publishers." Thx.